src/test/java/core/basesyntax/service/reader/CsvFileReaderTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.service.reader;
+
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CsvFileReaderTest {
+    private static FileReader fileReader;
+    private static final String VALID_PATH = "src/test/resources/input.csv";
+    private static final String INVALID_PATH = "in/valid/path/input.csv";
```

Comment:
Get your idea, however probably that would be better to generate like random string for representing invalid path